hourglass.py

```diff
@@ -87,14 +90,14 @@ def _start_train(self, batch_size, model_base_dir, epochs, initial_epoch, model_
         self.model.fit_generator(generator=train_generator, validation_data=val_generator, steps_per_epoch=len(train_generator), \
             validation_steps=len(val_generator), epochs=epochs, initial_epoch=initial_epoch, callbacks=callbacks)
 
-    def train(self, batch_size, model_save_base_dir, epochs):
+    def train(self, batch_size, model_save_base_dir, epochs, subset):
         current_time = datetime.today().strftime('%Y-%m-%d-%Hh-%Mm')
 
         model_subdir = current_time + '_batchsize_' + str(batch_size) + '_hg_' + str(self.num_stacks)
 
-        self._start_train(batch_size=batch_size, model_base_dir=model_save_base_dir, epochs=epochs, initial_epoch=0, model_subdir=model_subdir, current_time=current_time)
+        self._start_train(batch_size=batch_size, model_base_dir=model_save_base_dir, epochs=epochs, initial_epoch=0, model_subdir=model_subdir, current_time=current_time, subset=subset)
     
-    def resume_train(self, batch_size, model_save_base_dir, model_json, model_weights, init_epoch, epochs, resume_subdir):
+    def resume_train(self, batch_size, model_save_base_dir, model_json, model_weights, init_epoch, epochs, resume_subdir, subset):
```

Comment:
this may be a problem later, but resuming training and having the model select a different subset might be an issue. But since it's for quick iterations, maybe it's not something to worry about

Reply:
Good point, we definitely cannot use subset for robust/repeatable results. I think saving the subset ids to disk so can resume can use the same set is ~probably~ overkill for now since this feature is just to make quick tests. I will add comments somewhere so we remember this flaw.

Reply:
Yeah, I think it's overkill too since it's just seeing quick improvements to any changes we make. Comments sound good :)